### DISCUSSION TITLE
Cancelling a plan is IMPOSSIBLE

### DISCUSSION THREAD

**Discussion by duncanjw** [vercel/community#4904] [Account Issue] [answered]
### Question

Hi. I was responsible for a team which I wanted to gracefully cancel letting its plan run until the end of its current billing period. This is a standard option for pretty much every subscription I've dealt with.

However, when I tried to do this I was told the old way to cancel the plan associated with my team was to DELETE it which was effective immediately.

It vanished along with its billing history etc. so I have no way of knowing whether the plan was cancelled and my credit card details removed. No acknowledgement either.

Ironically I was actually sunsetting this team as I was switch to a new company and needed to start the equivalent team but under a different account but I am unimpressed with the way the cancellation was handled as effectively I am out of pocket.

**Comment by duncanjw** [comment]
I couldn't even raise this with Vercel support since having deleted the only team I was administering I has reduced to a hobbyist and my attempt to raise a case simply pointed me back at the community. Very poor UX.

**Comment by amyegan** [comment, marked as answer]
Thanks for sharing this feedback both here and in your support case. I wanted to share that we've improved the process of downgrading a team from pro to hobby. 

Full steps and more information can be found at [vercel.com/docs/accounts/plans/hobby#downgrading-to-hobby](https://vercel.com/docs/accounts/plans/hobby#downgrading-to-hobby)
